### PR TITLE
Update map marker colors and map size

### PR DESCRIPTION
**Pull request by kaohman**
I updated the map markers to display different colors if they are in the visited or bucket list. If they are in both, right now the visited list color will take precedent. 

I also updated the map size on the page so that the state display is more easily visible - this required changing the park pop-ups to be smaller as well.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -151,8 +151,8 @@ class App extends Component {
               />
               <div className="filters">
                 <button onClick={this.showAllParks}>Show All Parks</button>
-                <button onClick={this.showVisitedParks}><i class="fas fa-hiking" id="partOfList"></i> Show {this.state.visitedParkCodes.length} Visited Parks</button>
-                <button onClick={this.showBucketList}><i class="fas fa-clipboard-list" id="partOfList"></i> Show {this.state.bucketListParkCodes.length} Bucket List Parks</button>
+                <button onClick={this.showVisitedParks}><img className="loc-icons" src="./assets/marker-icon-green.png" alt="green icon" /> Show {this.state.visitedParkCodes.length} Visited Parks</button>
```
Comment (trbachmann) [inline src/App.js:154]: The purple and green icons look great on the map and I like how cohesive it looks with them on the view buttons also

File: src/FilterControls.js [hunk 2]
```diff
@@ -25,14 +25,16 @@ class FilterControls extends Component {
...
       <div className="filter-controls">
-        <select id="select-menu" value={this.props.stateName} onChange={this.getState}>
-          <option value="default">Please pick a state</option>
-        {
-          Object.keys(this.props.usStates).map(usState => {
-            return(<option value={[usState]} key={this.props.usStates[usState].abbreviation}>{[usState]}</option>)
-          })
-        }
-        </select>
+        <label className="filter-label">View Parks By State:
```
Comment (trbachmann) [inline src/FilterControls.js:28]: Good idea, thank you for adding this label for View by state

File: src/ParkMap.js [hunk 3]
```diff
@@ -50,11 +75,26 @@ class ParkMap extends Component {
...
+                  </Marker>
+                )
+              } else if (this.props.bucketListParks.includes(park.urlCode)) {
+                return (
+                  <Marker position={[lat, lon]} icon={violetIcon} onClick={this.getPark} key={park.urlCode} id={park.urlCode}>
+                    <Tooltip>{park.parkName}</Tooltip>
+                  </Marker>
+                )
+              } else {
+                console.log(park.urlCode)
```
Comment (trbachmann) [inline src/ParkMap.js]: Will you please delete the console.log?

File: src/styles/variables.scss [hunk 4]
```diff
@@ -1,4 +1,4 @@
-$map-height: 500px;
-$map-width: 80%;
+$map-height: 450px;
```
Comment (trbachmann) [inline src/styles/variables.scss:1]: Thank you for updating the map size! It looks good and the state information is more eye catching now when a user chooses a state

File: src/ParkMap.js [hunk 5]
```diff
@@ -50,11 +75,26 @@ class ParkMap extends Component {
             this.props.parks.map(park => {
               let lat = (park.latitude.includes('N')) ? park.latitude.replace(/.N$/, '') : park.latitude.replace(/.S$/, '').replace(/^/, '-');
               let lon = (park.longitude.includes('E')) ? park.longitude.replace(/.E$/, '') : park.longitude.replace(/.W$/, '').replace(/^/, '-');
-              return(
-                <Marker position={[lat, lon]} onClick={this.getPark} key={park.urlCode} id={park.urlCode}>
-                  <Tooltip>{park.parkName}</Tooltip>
-                </Marker>
-              )
+              if (this.props.visitedParks.includes(park.urlCode)) {
```
Comment (trbachmann) [inline src/ParkMap.js:78]: Awesome job getting the different icon colors to display when it's a visited or bucket list park! Really like how the map looks now when a user has visited and bucket list parks

**Review by trbachmann — APPROVED** [review]
Awesome work on getting the different color icons to display for visited and bucket list parks! I really like how the map looks now when a user has some visited and bucket list parks. Also the view buttons look really cohesive with the map now.

File: src/styles/variables.scss (hunk 4 above)
Comment (AdamN8142) [reply src/styles/variables.scss:1]: Definitely like this size

**Comment by AdamN8142** [comment]
Great job, i think this is nice attention to detail. Now the user can look at the map with all parks and easily recognize where they have been and where they could potentially go.

**Pull request merged**